### ISSUE TITLE
Adding 'differentiable' section to each function docstring

### ISSUE THREAD

**Issue by tomer196** [pytorch/pytorch#61513]
## 🚀 Feature

Adding 'differentiable' section to each function docstring.

## Motivation

Many times I find my self wonder if specific function is differentiable or not. It can also be the place to describe other autograd issues, for example: 
- Differentiable withe respect to which one of the inputs.
- Twice differentiable.  
- Any related issues.

## Pitch

Each function docstring contain 'Differentiable' section

**Comment by vadimkantorov** [comment]
Also would love this! Especially for torch.full, torch.lerp and loss functions (differentiability wrt to targets)

**Comment by heitorschueroff** [comment]
Adding to triage review to discuss as this could be a very interesting addition to our documentation

**Comment by albanD** [comment]
I feel like a "not differentiable" would be more concise because most functions that are differentiable actually work. Only a few loss functions wrt target or more complicated special functions don't have formulas (yet).
